docs/README.md

```diff
@@ -1,3 +0,0 @@
-# single-spa documentation
```

Comment:
The documentation was moved away from Github a few major releases ago - these files were kept here for old links. I don't think many old links to these pages are still being clicked on, so I removed the docs and examples folders